backend/internal/controllers/book_test.go

```diff
@@ -50,3 +52,155 @@ func TestGetNullBook(t *testing.T) {
 
 	require.Equal(t, "book not found", response)
 }
+
+// Test to just create the book
+func TestJustCreateBook(t *testing.T) {
+	body := `{"title": "Lonely Book", "author":"Lonely Man", "image":null}`
+	testutils.MakeHttpRequest("POST", ts.URL+"/books", body, http.StatusCreated,
+		nil, t)
+
+	var response []models.Book
+
+	// get req should not cause an error if book exists without any languages
+	testutils.MakeHttpRequest("GET", ts.URL+"/books", "", 200, &response, t)
+
+	require.Len(t, response, 6)
+}
+
+// Test creating a book and its contents together
+func TestCreateBookandBookDetails(t *testing.T) {
+	body := `{"title": "Harry Potter", "author":"JK Rowling", "image":null}`
+	var createdBook models.Book
+	testutils.MakeHttpRequest("POST", ts.URL+"/books", body, http.StatusCreated,
+		&createdBook, t)
+
+	require.Equal(t, "Harry Potter", createdBook.Title)
+	require.Equal(t, "JK Rowling", createdBook.Author)
+	require.Equal(t, []string{}, createdBook.Languages)
+
+	body = `{"language": "en", 
+	"Read": {"video": null, "body":"hp_rb"}, 
+	"Explore": {"video": null, "body":"hp_eb"},  
+	"Learn": {"video": null, "body":"hp_lb"}}`
```

Comment:
Read/Explore/Learn should be lowercased since that's what's in the `json:` field tags. Noticed this in a few other tests down below too